src/CoreWCF.WebHttp/src/CoreWCF/Description/WebHttpEndpoint.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using CoreWCF.Web;
+
+namespace CoreWCF.Description
+{
+    public class WebHttpEndpoint : WebServiceEndpoint
+    {
```

Comment:
This class and the base class aren't needed. They are used in ways which aren't applicable to CoreWCF. In WCF you normally call ServiceHost.AddServiceEndpoint and specify the type, binding and contract. There's also an overload which takes a ServiceEndpoint which is the OM class created to hold those 3 pieces of info when you call the overload with 3 params. This class is sued to use the ServiceEndpoint overload and preconfigures the endpoint to use WebHttpBinding, as well as exposing lots of helper properties to configure the behavior. We don't have an overload which takes a ServiceEndpoint instance so these classes can't be used.

Reply:
This was addressed in 063bb16c3698777ec49fc33d40cd832d952fd106.